docs/extensibility-provider-api.json

```diff
@@ -0,0 +1,490 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Deployments Extensibility Provider API",
+    "description": "Deployments extensibility provider OpenAPI specification.",
+    "version": "v1"
+  },
+  "paths": {
+    "/save": {
+      "post": {
+        "tags": ["Resource operations"],
+        "summary": "Save operation",
+        "description": "Saves an extensible resource",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/ExtensibilityOperationRequest"
+              },
+              "example": {
+                "Import": {
```

Comment:
Are the well defined properties in the contract case-insensitive? For documentation, it would be good to stick to the convention of camel case rather than pascal case.

Reply:
Good catch. I'm using `Swashbuckle.AspNetCore.Filters` to generate the examples, and it turns out it only uses JSON serializer options that are configured with `services.AddControllers()`, so I updated the dev host to use controllers, which fixes the casing issue.